vms/platformvm/vm_regression_test.go

```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/ava-labs/avalanchego/utils/constants"
```

Comment:
I'm trying to make it easy to write tests - so that then we can write good tests... Admittedly, this file specifically includes edge case testing that at one point (either in a PR, in test, or in prod) was failing unexpected... So they often have a pretty specific set of operations...